webapp/.nvmrc

```diff
@@ -0,0 +1 @@
+21.1.0
```

Comment:
Some JS devs have a script to load these .nvmrc files and set the correct node version automatically. I also updated the package.json file with the correct node engine version.